ephys.py

```diff
@@ -0,0 +1,113 @@
+import datajoint as dj
+
+import actions
+import equipment
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+
+@schema
+class BrainLocation(dj.Manual):
+    # XXX: not connected to reference.*
+    # <class 'electrophysiology.models.BaseBrainLocation'>
+    definition = """
+    ccf_ap:			float		# ccf ap
+    ccf_dv:			float		# ccf dv
+    ccf_lr:			float		# ccf lr
+    ---
+    allen_ontology:		varchar(255)	# allen ontology
+    json:			varchar(255)	# json
```

Comment:
if ccf ap/dv/lr doesn't make sense to you, it doesn't make sense to me either :)

Reply:
Ah, I actually was referring to the `json` field. The `ccf` stands for common coordinate framework - a specification to describe mouse brain coordinates established by Allen Institute (http://help.brain-map.org/display/mousebrain/API). `ap` = anterior-vs-posterior, `dv`=dorsal-vs-ventral (same as superior-vs-inferior in the picture, respectively) and `lr`=left-vs-right.

![image](https://user-images.githubusercontent.com/3084661/32133128-44db41bc-bb97-11e7-808a-d6850c085113.png)

Reply:
ah yes, the original extended the base django object so that everything had a UUID pkey and json field, which were included in the machine translation. we are zapping in the 'dj-native' copy. Good catch - will grep for any other missed ones before updating.